### ISSUE TITLE
chat without login?

### ISSUE THREAD

**Issue by Rajan** [fstgerm/meteor-chat#1]
Is it possible to chat without logging in?
If yes, do we need to make any changes?

**Comment by fstgerm** [comment]
Yes, you can use an _account guest_ package that does not require a login. 

https://atmospherejs.com/artwells/accounts-guest
